StreamChatUITestsAppUITests/Robots/ParticipantRobot.swift

```diff
@@ -88,6 +88,21 @@ final class ParticipantRobot: Robot {
         )
         return self
     }
+
+    /// The given text will be decorated with the index, eg "message-10"
+    @discardableResult
+    func sendMultipleMessages(repeatingText text: String, count: Int) -> Self {
+        var texts = [String]()
+        for index in 1...count {
+            texts.append("\(text)-\(index)")
+        }
+
+        texts.forEach {
+            sendMessage($0)
+            chill(duration: 100)
```

Comment:
This doesn't seem very efficient, why don't we just wait for the last message to appear? 
```suggestion
            waitForTextMessage(texts.last)
```